src/renderer/src/components/LeaveProjectModal/LeaveProjectConfirmation.tsx

```diff
@@ -0,0 +1,139 @@
+import { Checkbox, Typography } from '@mui/material'
+import ErrorIcon from '@mui/icons-material/Error'
+import WarningAmberIcon from '@mui/icons-material/WarningAmber'
+import { Column, Row } from '../LayoutComponents'
+import { defineMessages, useIntl } from 'react-intl'
+import { Button } from '../Button'
+import { MEDIA, OBSERVATIONS } from '@renderer/lib/Observations'
+import { useState } from 'react'
+import { theme } from '@renderer/theme'
+import { useMapeoDeviceStore } from '@renderer/hooks/stores/mapeoDeviceStore'
+
+const m = defineMessages({
+  leaveConfirmation: {
+    id: 'components.LeaveProjectModal.leaveConfirmation',
+    defaultMessage: 'Are you sure you want to leave project {projectName}',
+  },
+  deleteWarning: {
+    id: 'components.LeaveProjectModal.deleteWarning',
+    defaultMessage:
+      'This will delete {observations} observations and {mediaSize}mb of data from this device.',
+  },
+  unsyncedWarningTitle: {
+    id: 'components.LeaveProjectModal.unsyncedWarningTitle',
+    defaultMessage: '5 unsynced changes',
```

Comment:
just to check, we're okay with this being a set number?

Reply:
yeah, i did that on purpose. seems unnecessary to set a random number or a number derived from observations